src/app/components/elements/CareersBanner.tsx

```diff
@@ -19,50 +21,92 @@ const CsAtWorkDescription = () => {
-            height="100%"
-            src={`https://www.youtube-nocookie.com/embed/${videoId}?enablejsapi=1&fs=1&modestbranding=1`}
-            allowFullScreen
-          />
-        </Col>
-        <Col className="video-description-column">
-          <h4>{isStudent(user) ? "Linking computer science to the real world" : "Computer Science at work"}</h4>
-          <CsAtWorkDescription />
-          <Button tag={Link} size="lg" className="mt-2" to={"/careers_in_computer_science"}>
-            See more careers videos
-          </Button>
-        </Col>
-      </Row>
-      <Row className="cs-journey">
-        <Col xs="auto" className="left-column">
-          <h4>
-            <a href="/pages/computer_science_journeys_gallery" rel="noopener noreferrer">
-              Computer Science Journeys
-            </a>
-          </h4>
-          <p className="mb-3">
-            Discover our monthly interview series and learn from passionate educators within the Isaac community, and
-            recently-graduated computer scientists who are doing AMAZING things in a huge range of computing-related
-            fields!
-          </p>
-        </Col>
-        <Col xs="auto" className="right-column">
-          <img src="/assets/cs_journeys.png" alt="cs journeys" />
-        </Col>
-      </Row>
-    </Container>
+    <div className="career-background-img">
+      <Container className="career-section">
+        <h4 className="career-title">Careers in Computer Science</h4>
+        <Row className="career-media-row">
+          <Col>
+            <CareerCard
+              imgSrc="/assets/cs_journeys.svg"
+              imgAlt="cs journeys"
+              title="Computer Science Journeys"
+              text="Discover our interview series and learn from passionate educators within the Isaac community, and recently graduated computer scientists, who are doing amazing things in a huge range of computing-related fields."
+              linkTo="/pages/computer_science_journeys_gallery"
+              linkText="Read our interviews"
+            />
+          </Col>
+          <Col>
+            <CareerCard
+              title={isStudent(user) ? "Linking computer science to the real world" : "Computer Science at work"}
+              text={<CsAtWorkDescription />}
+              linkTo="/careers_in_computer_science"
+              linkText="See more career videos"
+            >
+              <div className="career-media-row-column">
+                <iframe
+                  title="career-video"
+                  className="career-media-row-video no-border"
+                  id="ytplayer"
+                  width="100%"
+                  height="100%"
+                  src={`https://www.youtube-nocookie.com/embed/${videoId}?enablejsapi=1&fs=1&modestbranding=1`}
+                  allowFullScreen
+                />
+              </div>
+            </CareerCard>
+          </Col>
+        </Row>
+        <div className="career-comment">
```

Comment:
can we adjust the width / margisn on this bottom comment section? It's slightly off centre on mobile: 

![image](https://github.com/user-attachments/assets/1ab24d4f-1ca3-4eda-bd58-a9cd71e1d0bc)